src/ui/components/opinions/Listaopinion_style.js

```diff
@@ -0,0 +1,27 @@
+import { makeStyles } from "@material-ui/core";
+import mq from "../../../config/mq";
+const useStyles = makeStyles((theme) => ({
+    containersx:{ 
+        height:'673px',
+        width:'1176px',
+        padding:'0px',
+        margin:'0px',
+    },
+    sx:{
+
+        display: 'flex',
+        height: '673px',
+        width:'912px',
+        borderRadius:'0.75rem',
+        backgroundColor:'#FFFFFF',
```

Comment:
the color must have  come from the main Theme file something like this theme.palette.color